src/components/App.tsx

```diff
@@ -1765,6 +1765,7 @@ class App extends React.Component<AppProps, AppState> {
       if (event.key === KEYS.SPACE && gesture.pointers.size === 0) {
         isHoldingSpace = true;
         setCursor(this.canvas, CURSOR_TYPE.GRABBING);
+        event.preventDefault();
```

Comment:
technically we should prevent only when `handleKeyboardGlobally == false`, but for this use case it should be fine